### ISSUE TITLE
[web][a11y] Web cannot enable accessibility when the page has a focused autofillable text element

### ISSUE THREAD

**Issue by nturgut** [flutter/flutter#60402]
Observed both in Desktop Chrome with VoiceOver. Text field gets VoiceOver focus in the web area, therefore user is not able to `enable accessibility button`.

```
[✓] Flutter (Channel unknown, 1.20.0-3.0.pre.76, on Mac OS X 10.15.4 19E287, locale en-US)
    • Flutter version 1.20.0-3.0.pre.76 at /Users/nurhan/development/flutter
    • Framework revision fd7a72ee6f (49 minutes ago), 2020-06-26 15:18:02 -0700
    • Engine revision fc0e27210c
    • Dart version 2.9.0 (build 2.9.0-19.0.dev 63cf56d925)

 
[!] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/nurhan/Library/Android/sdk
    • Platform android-29, build-tools 29.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] VS Code (version 1.46.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.12.0

[✓] Connected device (4 available)
    • sdk gphone x86 • emulator-5554                        • android-x86    • Android 11 (API 30) (emulator)
    • iPhone 11 Pro  • 82A0BF3E-8596-4AE4-BFA3-4E5E45FE2CF3 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-13-0 (simulator)
    • Web Server     • web-server                           • web-javascript • Flutter Tools
    • Chrome         • chrome                               • web-javascript • Google Chrome 83.0.4103.116
```

**Comment by nturgut** [comment]
This worked when tested with gallery code (when I locally run it with the current engine).

It didn't work with the example https://gist.github.com/nturgut/de835f60afe83d945d2573ab5ddd375e

Therefore, I suspect, using `form` element for implementing Autofill caused the issue.

**Comment by chunhtai** [comment]
some update: @nturgut  will get back to this issue once they has time.

**Comment by TahaTesser** [comment]
<details> 
<summary>flutter doctor -v</summary> 

```bash
[✓] Flutter (Channel master, 1.22.0-10.0.pre.251, on Mac OS X 10.15.6 19G2021
    x86_64, locale en-GB)
    • Flutter version 1.22.0-10.0.pre.251 at
      /Users/tahatesser/Code/flutter_master
    • Framework revision d26268bb9e (13 hours ago), 2020-09-18 09:06:26 +0900
    • Engine revision 2abe69c608
    • Dart version 2.10.0 (build 2.10.0-136.0.dev)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/tahatesser/Code/sdk
    • Platform android-30, build-tools 30.0.2
    • ANDROID_HOME = /Users/tahatesser/Code/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.0, Build version 12A7209
    • CocoaPods version 1.9.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 49.0.2
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.49.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.14.1

[✓] Connected device (3 available)
    • macOS (desktop)  • macos      • darwin-x64     • Mac OS X 10.15.6 19G2021
      x86_64
    • Web Server (web) • web-server • web-javascript • Flutter Tools
    • Chrome (web)     • chrome     • web-javascript • Google Chrome
      85.0.4183.102

• No issues found!
```

</details>
